src/Features/Core/Portable/MoveToNamespace/MoveToNamespaceOptionsResult.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+
+namespace Microsoft.CodeAnalysis.MoveToNamespace
+{
+    internal class MoveToNamespaceOptionsResult
+    {
+        public static readonly MoveToNamespaceOptionsResult Cancelled = new MoveToNamespaceOptionsResult(isCancelled: true);
+
+        public bool IsCancelled { get; }
+        public string Namespace { get; }
+
+        public MoveToNamespaceOptionsResult(bool isCancelled)
+        {
+            IsCancelled = true;
```

Comment:
So I can pass (isCancelled: false) and it marks it as cancelled? Maybe have a static instance `MoveToNamespaceOptionsResult.Cancelled` and then don't expose this constructor?

Reply:
Good call